### PR TITLE
Add mgmt command for (re)calculation of daily costs per single day.

### PR DESCRIPTION
**Pull request by xor-xor**

### CODE REVIEW

File: src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        return CostDateStatus.objects.filter(
+            date=date_,
+            **{'forecast_calculated' if forecast else 'calculated': True}
+        ).values_list('date', flat=True).exists()
+
+    def _has_accepted_costs(self, date_, forecast):
+        return CostDateStatus.objects.filter(
+            date=date_,
+            **{'forecast_accepted' if forecast else 'accepted': True}
+        ).values_list('date', flat=True).exists()
```
Comment (mkurek) [inline src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py]: no need for `values_list` to check if any record `exists`

File: src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            )
+            return
+
+        # Determine which plugins are needed and finally calculate & save the
+        # costs.
+        plugins = []
+        for p in collector.get_plugins():
+            if p.name in pricing_service_names_verified:
+                plugins.append(p)
+        processed_costs = collector.process(date_, forecast, plugins=plugins)
```
Comment (mkurek) [inline src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py]: we could think about wrapping these 3 calls to collector into some Collector's method, which will have date (start and end?), forecast and plugins param and will do the rest - @xor-xor what do you think?

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py (hunk 1 above)
Comment (xor-xor) [reply src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py]: Ah, right - I'm a copy & paste victim :wink:

File: src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py (hunk 2 above)
Comment (xor-xor) [reply src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py]: Makes sense, but only if this method will be used anywhere else apart from this mgmt command - do you see any use for it in the future..?

On the other hand, I see that `Collector` already has methods like `process_period` and`process`, so adding another one, with similar name/behavior can be confusing to ppl working with it later.

Comment (xor-xor) [reply src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py]: Done.

**Review by mkurek — APPROVED** [review]

File: src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        query_params = {
+            'active': True,
+            'plugin_type': PricingServicePlugin.pricing_service_fixed_price_plugin.id,  # noqa: E501
+        }
+        # ...but when `-p` option is present, take only PricingServices
+        # specified with it.
+        if len(pricing_service_names) > 0:
+            query_params.update({'name__in': pricing_service_names})
+        pricing_services = PricingService.objects.filter(**query_params)
+        pricing_service_names_verified = [ps.name for ps in pricing_services]
```
Comment (szok) [inline src/ralph_scrooge/management/commands/scrooge_calculate_dailycosts.py]: Why not
pricing_services.vlaues_list('name', flat=True)
?

**Review by szok — APPROVED** [review]

**Pull request merged**